### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by ThinkFar** [lxdware/lxd-dashboard#72]
…Title

**Comment by matthewalanpenning** [comment]
Yes it is. However, recently all of the new development efforts have been focused on my python port of the dashboard https://github.com/PenningLabs/lxconsole

**Comment by ThinkFar** [comment]
Good to know! 
How does it compare to this one in terms of stability and using it in production?

**Comment by matthewalanpenning** [comment]
The python version is currently labeled as beta release primary because there are a lot of new changes still being pushed out to it, and the layout or design may change with new added features. But I do test the software quite a bit before pushing out updates. In both the Python and PHP versions I don't yet have security features to enforce password lengths or complexity or multi-factor authentication methods, so I would recommend not exposing the web UI to the public Internet.

**Comment by ThinkFar** [comment]
Alright, is this just in regarding the python port of the dashboard or both?

**Comment by ithakaa** [comment]
Amazing work !!!
